### PR TITLE
PR for removing functions which overlap with jQuery

### PR DESCRIPTION
**Pull request by ShashankaNataraj**
1. Removed getViewportHeight() and used $(window).height()
2. corrected extra commas

https://github.com/h5bp/Effeckt.css/issues/312

### CODE REVIEW

**Comment by ShashankaNataraj** [comment]
@grayghostvisuals  @arthurvr does this change look OK or are there any changes you would like to see with this?

File: js/demo/lazy-loading.js [hunk 1]
```diff
@@ -26,14 +26,15 @@ var EffecktLazyLoading = {
 
   bindUIActions: function() {
 
-    var self = this;
+    var self = this,
+      windowObj = $(window);
```
Comment (arthurvr) [inline js/demo/lazy-loading.js]: I don't think we have much of a naming convention for jQuery/non-jQuery variables here (it at least isn't in the style guide) but a name like `windowObj` makes me think it's just `window`. What does @h5bp/effeckt-css think?

**Comment by arthurvr** [comment]
Reminds me that one day we really need unit tests as well. Not really related to this PR though.

File: js/demo/lazy-loading.js (hunk 1 above)
Comment (grayghostvisuals) [inline js/demo/lazy-loading.js]: using ``window`` instead of ``windowObj`` is my preference

Comment (ShashankaNataraj) [inline js/demo/lazy-loading.js]: Since $(window) was being used twice, I thought of creating a variable and assigning $(window) to that. Will it be OK if I use $(window) twice here?

File: js/demo/lazy-loading.js [hunk 2]
```diff
@@ -105,8 +93,8 @@ var EffecktLazyLoading = {
         self._scrollPage();
       }, 1000 / 60);
     }
-  },
+  }
```
Comment (arthurvr) [inline js/demo/lazy-loading.js]: Let's not handle this kind of things in unrelated commits. We can cleanup all the things when handling https://github.com/h5bp/Effeckt.css/issues/314.

Comment (ShashankaNataraj) [inline js/demo/lazy-loading.js]: @arthurvr Sure alright, will close this PR and open another one with the changes mentioned above. Also, once we add a jshint file, we should open an issue for jshint compliance.

**Comment by arthurvr** [comment]
@ShashankaNataraj Please don't create a new PR just to do updates to an existing one ;) Amend your commits here (or add new commits) but don't create the extra noise. It's also hard to track down later on.

**Comment by ShashankaNataraj** [comment]
@arthurvr alright, will do, thanks for being patient.

**Comment by ShashankaNataraj** [comment]
@arthurvr @grayghostvisuals OK guys, will this do?

File: js/demo/lazy-loading.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ var EffecktLazyLoading = {
   _scrollPage: function() {
 
     var self = this;
-    
+
```
Comment (arthurvr) [inline js/demo/lazy-loading.js]: These changes are perfectly fine but they don't deserve here. This PR is meant to fix a specific ticket and shouldn't touch anything more. Same thing as https://github.com/h5bp/Effeckt.css/pull/313#discussion_r31379462. Let's keep the history/diffs clean and on-topic.

File: js/demo/lazy-loading.js [hunk 4]
```diff
@@ -74,7 +74,6 @@ var EffecktLazyLoading = {
       scrolled = $('.effeckt-page-active').scrollTop(),
       viewed = scrolled + this.viewportHeight,
       elTop = $(el).offset().top,
-      elBottom = elTop + elHeight,
```
Comment (arthurvr) [inline js/demo/lazy-loading.js]: Same thing as above. This is IMHO worse at it's more than just a code style tweak.

File: js/demo/lazy-loading.js [hunk 5]
```diff
@@ -109,4 +96,4 @@ var EffecktLazyLoading = {
 
 };
 
-EffecktLazyLoading.init();
\ No newline at end of file
+EffecktLazyLoading.init(window);
```
Comment (arthurvr) [inline js/demo/lazy-loading.js]: Let's be consistent with the [other demo page](https://github.com/h5bp/Effeckt.css/blob/master/js/demo/demo.js) and not pass this for the time being, or pass it in for both. It isn't really needed IMHO.

**Comment by arthurvr** [comment]
Just to help our future selves this was closed because of the discussion in https://github.com/h5bp/Effeckt.css/issues/312.
